File: Pyvvo.Logistics.Model/Model/OrderLineItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Pyvvo.Logistics.Model
{
    public class OrderLineItem
    {
        [Key, Required] public long Id { get; set; }
        [Required] public long OrderId { get; set; }
        [Required] public long FulfillmentSatusId { get; set; }
        [Required] public long VariantId { get; set; }
        [Required] public DateTime CreatedOn { get; set; }
        public DateTime UpdatedOn { get; set; }
        public int LineNumber { get; set; }
        public string Name { get; set; }
        public bool IsOutOfStock { get; set; }
        public int Quantity { get; set; }
        public double TotalPrice { get; set; }
        public Order Order { get; set; }
        public Status FulfillmentSatus { get; set; }
        public Variant Variant { get; set; }
        public List<InvoiceLineItem> InvoiceLineItems { get; set; }
        public List<RefundLineItem> RefundLineItems { get; set; }
        public List<ShipmentLineItem> ShipmentLineItems { get; set; }
        public List<ProcessingLineItem> ProcessingLineItems { get; set; }
        public List<ReturnLineItem> ReturnLineItems { get; set; }
    }
}

File: Pyvvo.Logistics/ClientApp/src/app/service/Model/order-line-item.ts
import { Order } from "./order";
import { Status } from "./status";
import { Variant } from "./variant";
import { InvoiceLineItem } from "./invoice-line-item";
import { RefundLineItem } from "./refund-line-item";
import { ReturnLineItem } from "./return-line-item";
import { ProcessingLineItem } from "./processing-line-item";
import { ShipmentLineItem } from "./shipment-line-item";

export class OrderLineItem {
  id: number;
  checked: boolean;
  disabled: boolean;
  createdOn: Date;
  updateOn: Date;
  lineNumber: number;
  name: string;
  isOutOfStock: boolean;
  quantity: number;
  maxQuantity: number;
  totalPrice: number;
  order: Order;
  fulFillmentStatus: Status;
  variant: Variant;
  invoiceLineItems: InvoiceLineItem[];
  refundLineItems: RefundLineItem[];
  shippmentLineItem: ShipmentLineItem[];
  processionLineItem: ProcessingLineItem[];
  returnLineItems: ReturnLineItem[];
}
